components/calendly_v2/calendly_v2.app.mjs

```diff
@@ -1,11 +1,213 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "calendly_v2",
-  propDefinitions: {},
+  propDefinitions: {
+    user: {
+      type: "string",
+      label: "User UUID",
+      description: "An user UUID",
+    },
+    eventId: {
+      type: "string",
+      label: "Event ID",
+      description: "An event UUID",
+      async options({ prevContext }) {
+        return await this._makeAsyncOptionsRequest({
+          prevContext,
+          requestType: "listEvents",
+        });
+      },
+    },
+    eventType: {
+      type: "string",
+      label: "Event Type",
+      description: "An event type UUID",
+      async options({ prevContext }) {
+        return await this._makeAsyncOptionsRequest({
+          prevContext,
+          requestType: "listEventTypes",
+        });
+      },
+    },
+    inviteeEmail: {
+      type: "string",
+      label: "Inviteee Email",
+      description: "The invitee's email",
+      optional: true,
+    },
+    status: {
+      type: "string",
+      label: "Event Status",
+      description: "Whether the scheduled event is `active` or `canceled`",
+      optional: true,
+      options: [
+        "active",
+        "canceled",
+      ],
+    },
+    maxEventCount: {
+      type: "integer",
+      label: "Max Event Count",
+      description: "The max number of events that can be scheduled using this scheduling link",
+    },
+    paginate: {
+      type: "boolean",
+      label: "Paginate",
+      description: "Whether to paginate or not",
+      optional: true,
+    },
```

Comment:
We should probably set `paginate` to be `true` by default.
```suggestion
    paginate: {
      type: "boolean",
      label: "Paginate",
      description: "Whether to paginate or not",
      optional: true,
      default: true,
    },
```

Reply:
Issue addressed